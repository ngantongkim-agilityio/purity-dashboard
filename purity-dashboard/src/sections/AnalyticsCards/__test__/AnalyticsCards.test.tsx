import { render } from '@testing-library/react';

// Components
import { AnalyticsCards } from '..';

jest.mock('@/services/analytics.ts', () => ({
  fetchAnalytics: jest.fn()
}));

jest.mock('@/services/products.ts', () => ({
  fetchLatestProducts: jest.fn()
}));

describe('AnalyticsCards component', () => {
  it('Should render AnalyticsCards component with default props', () => {
    const { container } = render(<AnalyticsCards />);

    expect(container).toMatchSnapshot();
  });
});
