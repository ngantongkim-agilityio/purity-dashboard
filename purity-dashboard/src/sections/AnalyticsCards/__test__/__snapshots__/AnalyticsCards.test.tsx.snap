// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnalyticsCards component Should render AnalyticsCards component with default props 1`] = `<div />`;
